src/nodes/Navigation/NavigationInfo.js

```diff
@@ -62,6 +62,14 @@ x3dom.registerNodeType(
 
             /**
              * Specifies the view angle and height for helicopter mode and min/max rotation angle for turntable in ]0, PI[, starting from +y (0) down to -y (PI)
+             * [0] helicopter vertical view angle theta
+             * [1] helicopter height
+             * [2] min vertical view angle theta
+             * [3] max vertical view angle theta
+             * [4] turntable min orbit rotation angle phi (yaw)  (### needs verification)

```

Comment:
[4] and [5] is also general zoom min/max  if all 7 params are provided: https://github.com/x3dom/x3dom/blob/1f251fd37c760f3aac892cc2398ac8cec90c54c3/src/nodes/Navigation/modes/DefaultNavigation.js#L39

Reply:
This part looks weird in function `x3dom.TurntableNavigation.prototype.calcOrbit`:
- there is some irrelevant code
- typeParams[ 4/5 ] can be used either used as min/max or as max/min

```
x3dom.TurntableNavigation.prototype.calcOrbit = function ( view, alpha, beta, absolute )
{
    var navi = this.navi;

    view._up   = view._flyMat.e1();
    view._from = view._flyMat.e3();

    var offset = view._from.subtract( view._at );

    var phi,
        theta;

    if ( absolute == true )
    {
        phi = beta;
        theta = alpha;
    }
    else
    {
        // angle in xz-plane
        // irrelevant because phi get's replaced before usage
        phi = Math.atan2( offset.x, offset.z );

        // angle from y-axis
        // irrelevant because theta get's replaced before usage
        theta = Math.atan2( Math.sqrt( offset.x * offset.x + offset.z * offset.z ), offset.y );

        phi -= beta;
        theta -= alpha;
    }

    // clamp theta
    // minTheta = typeParams[ 2 ]; maxTheta = typeParams[ 3 ]
    theta = Math.max( navi._vf.typeParams[ 2 ], Math.min( navi._vf.typeParams[ 3 ], theta ) );

    if ( navi._vf.typeParams[ 4 ] <= navi._vf.typeParams[ 5 ] {
        // minPhi = params[4], maxPhi = param[5]
        phi = Math.max( navi._vf.typeParams[ 4 ], Math.min( navi._vf.typeParams[ 5 ], phi ) );
    }
    else
    {
        // maxPhi = params[4], minPhi = param[5]
        if      ( beta > 0 && phi < navi._vf.typeParams[ 4 ] && phi > navi._vf.typeParams[ 5 ] ) {
            // Let's take the greater value if phi is within the boundaries
            phi = navi._vf.typeParams[ 4 ];
         }
        else if ( beta < 0 && phi < navi._vf.typeParams[ 4 ] && phi > navi._vf.typeParams[ 5 ] ) {
            // Let's take the smaller value if phi is within the boundaries
            phi = navi._vf.typeParams[ 5 ];
        }
    }
[...]
```

In any case, I guess it would be better to replace these positional parameters with named attributes...

Until then: I guess concerning the documentation it's better to omit these details, therefore I take them out of this PR.